packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -959,6 +961,31 @@ export class Function extends FunctionBase {
     });
   }
 
+  /**
+   * Defines an alias for this function.
+   *
+   * The alias will automatically be updated to point to the latest version of
+   * the function as it is being updated during a deployment.
+   *
+   * ```ts
+   * declare const fn: lambda.Function;
+   *
+   * fn.addAlias('Live');
+   *
+   * // Is equivalent to
+   *
+   * new Alias(this, 'AliasLive', {
```

Comment:
```suggestion
   * new lambda.Alias(this, 'AliasLive', {
```

I'm anticipating this breaking rosetta the way it is written.